src/control/cam_comp.F90

```diff
@@ -75,8 +75,7 @@ subroutine cam_init(caseid, ctitle, model_doi_url,                         &
         eccen, obliqr, lambm0, mvelpp,                                        &
         perpetual_run, perpetual_ymd,                                         &
         dtime, start_ymd, start_tod, ref_ymd, ref_tod,                        &
-        stop_ymd, stop_tod, curr_ymd, curr_tod,                               &
-        cam_out, cam_in)
+        stop_ymd, stop_tod, curr_ymd, curr_tod)
```

Comment:
Can't figure out how to leave a comment in the right place, but can the `cam_in_t` and `cam_out_t` use statements on line 28 be removed since we're using cam_in and cam_out directly from physics_types?